### PR TITLE
Update UltimaJointGenotyping to use GATK 4.5.0.0 for filtering

### PR DESCRIPTION
**Pull request by meganshand**
Updating the VETS (filtering) pipeline for the Ultima Joint Genotyping pipeline to GATK 4.5.0.0. The PR with the rest of the GATK updates to 4.5.0.0 is coming soon, but this one is a bit separated since VETS had changed a bit so we had to rework the WDL to match.

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [x] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [x] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: pipelines/broad/dna_seq/germline/joint_genotyping/UltimaGenomics/UltimaGenomicsJointGenotyping.changelog.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 1.1.5
+2023-12-14 (Date of Last Commit)
+
+* Updated VETS filtering pipeline to GATK version 4.5.0.0.
```
Comment (jessicaway) [inline pipelines/broad/dna_seq/germline/joint_genotyping/UltimaGenomics/UltimaGenomicsJointGenotyping.changelog.md:4]: Does this affect outputs?

**Review by jessicaway — APPROVED** [review]

File: pipelines/broad/dna_seq/germline/joint_genotyping/UltimaGenomics/UltimaGenomicsJointGenotyping.changelog.md (hunk 1 above)
Comment (meganshand) [reply pipelines/broad/dna_seq/germline/joint_genotyping/UltimaGenomics/UltimaGenomicsJointGenotyping.changelog.md]: It shouldn't. I updated the comment here.

**Review by kayleemathews — COMMENTED** [review]
From a documentation perspective, I don't think there are any updates that need to be made here.

**Review by nikellepetrillo — APPROVED** [review]
looks good, thanks!

**Comment by nikellepetrillo** [comment]
rerunning the reblocking plumbing test here:  https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/65402/console

**Pull request merged**